src/main/java/bio/terra/service/filedata/flight/ingest/IngestUtils.java

```diff
@@ -1,25 +1,31 @@
 package bio.terra.service.filedata.flight.ingest;
 
-import bio.terra.service.filedata.flight.FileMapKeys;
-import bio.terra.service.resourcemanagement.google.GoogleBucketResource;
 import bio.terra.stairway.FlightContext;
 
 public final class IngestUtils {
   private IngestUtils() {}
 
-  public static GoogleBucketResource getBucketInfo(FlightContext context) {
-    // The bucket has been selected for this file. In the single file load case, the info
-    // is stored in the working map. In the bulk load case, the info is stored in the input
-    // parameters.
+  /**
+   * Given a {@link FlightContext} object, look to see if the there is a value in the input map and
+   * if not, read it from the working map
+   *
+   * @param context The FlightContext object to examine
+   * @param key The map key to attempt to read values from
+   * @param clazz Class used to deserialize the value from the map
+   * @param <T> The type of the expected value in the maps
+   * @return A typed value from the flight context with type T or null if no value is found
+   */
+  public static <T> T getContextValue(FlightContext context, String key, Class<T> clazz) {
```

Comment:
This seems like it should be in `FlightUtils` since it doesn't appear to be ingest specific. Also the comment refers to bucket / storage account but it's not clear that that's the only use for this method, or that the comment still applies to this code.

Reply:
Yeah, I was wondering about that.  The main use was for the ingest flight but it certainly seems more generic than that.  I'll move it there and get rid of this class